### ISSUE TITLE
Compilation issues with c++1y gcc 4.9.2

### ISSUE THREAD

**Issue by khlebnikov** [CGAL/cgal#83]
Hello,

I just wanted to report a problem when compiling some code using CGAL 4.6 with a -std=c++1y compiler switch with gcc 4.9.2.

The problem appeared in Line_3.h (in my case) at lines 78 and 83 saying that the get() call is ambiguous. It turned out to clash with std::get, so replacing get(base) with CGAL::get(base) fixed the issue.

Hope this is at least a bit helpful,
   Rostislav.

**Comment by khlebnikov** [comment]
Forgot to add that for me it is in Cartesian/Line_3.h as I am using a Simple_cartesian kernel.

**Comment by lrineau** [comment]
Have you tried with `-std=c++11`? `std::get` is already in C++11, and I do not understand what in C++1y could trigger that bug.

Small world effect: it might be that the bug you report is already fixed by the pull-request #78. That pull-request was initiated by another problem, not related to C++11 at all, but to `boost::get` (for property maps). The solution of the pull-request is to rename the `CGAL::get` to something that cannot conflict with anything in Boost or the STL.

Can you try the patch that is proposed in the pull-request?

**Comment by khlebnikov** [comment]
Hello Laurent,

yes, c++11 works. As far as I know, the c++14 introduces ```std::get``` to get tuple member by type as opposed to c++11's get by index and I assume this is what causes a problem with resolution.

I really think that this pull request will fix this issue because there will be no name conflict.

Rostislav.

PS: I guess it makes sense to close this issue.

**Comment by lrineau** [comment]
Your bug will be closed once the pull-request is merged into `cgal/master`. For the moment the pull-request in under testing in our private `integration` branch. But we have many such pull-requests, and there is always one every day that breaks the tests.

**Comment by lrineau** [comment]
I think that bug is now closed by #78.
